open_academy/models/course.py

```diff
@@ -8,7 +8,7 @@ class Course(models.Model):
     name = fields.Char(required=True)
     title = fields.Text()
     description = fields.Text()
-    responsible_user_id = fields.Many2one('res.users')
+    responsible_user_id = fields.Many2one('res.users', default=False)
```

Comment:
If I'm not wrong, the value is false by default itself, you don't need to specify this

Or why are you changing it?

```suggestion
    responsible_user_id = fields.Many2one('res.users')
```

Reply:
I was testing something and I missed to delete that attribute, sorry